docs/notifications.md

```diff
@@ -57,13 +57,36 @@ The email template uses variables to include dynamic content. These variables ar
  - `emailHelpdeskHref`: The URL for the helpdesk.
 
 #### Updating Templates at Runtime
-Thymeleaf supports runtime updates to email templates. This allows you to make changes without restarting the application, which is useful for quick updates or tweaks:
- - **Enable Template Cache Refresh:** By default, Thymeleaf caches templates for performance. To enable runtime updates, you need to configure Thymeleaf to refresh templates automatically. This can typically be done by setting the `spring.thymeleaf.cache` property to `false` in your application configuration file (`application.yml`):
+
+The Negotiator provides an [REST](REST.md) endpoint to update the email template at runtime. This is only enabled when the spring.thymeleaf.prefix is set in the `application.yml` file e.g.:
+
 ```yaml
-spring.thymeleaf.cache=false
+spring:
+  thymeleaf:
+    prefix: file:resources/templates/
+```
+Using the endpoint as an **admin** user, you can update the email template without restarting the application. This is useful for making quick changes or tweaks to the template. 
+The content of the email template can be updated by sending a POST request to the `/api/v3/email-templates/<template_name>` endpoint with the new template content in the request body. This endpoint is protected by the `ROLE_ADMIN` role. 
+To reset the template to the default content, send a POST request to the `/api/v3/email-templates/<template_name>/reset` endpoint.
+You can get a list of all available templates by sending a GET request to the `/api/v3/email-templates` endpoint. It is not possible to add new templates or delete existing ones.
+
+#### Example of a POST request to update the email template
+```bash
+curl -X POST \
+  http://localhost:8081/api/v3/email-templates/email-notification \
+  -H 'Content-Type: application/xml' \
+  -H 'Authorization: Bearer <token>' \
+  -d '<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+    <head>
+        <title>Email notification</title>
+    </head>
+    <body>
+    Hello <span th:utext="${recipient.getName()}"></span>
+    </body>
+</html>'
 ```
```

Comment:
we should update the documentation. No need to document the API endpoint when we have a UI, plus I would put it under admin documentation not dev.